sdk/schemaregistry/schema-registry-avro/test/errors.spec.ts

```diff
@@ -14,6 +15,30 @@ import { v4 as uuid } from "uuid";
 
 chaiUse(chaiPromises);
 
+async function assertError<T>(
```

Comment:
nit: rename since this assertion is specific to `AvroSerializationError`? 

```suggestion
async function assertSerializationError<T>(
```